doc/man3/SSL_read_early_data.pod

```diff
@@ -186,6 +199,19 @@ In the event that the current maximum early data setting for the server is
 different to that originally specified in a session that a client is resuming
 with then the lower of the two values will apply.
 
+Some server applications may wish to have more control over whether early data
+is accepted or not. For example to mitigate replay risks (see L</REPLAY PROTECTION>
+below). The functions SSL_CTX_set_allow_early_data_cb and
+SSL_set_allow_early_data_cb set a callback which is called at a point in the
+handshake immediately before a decision is made to accept or reject early data.
+The callback is provided with a pointer to the user data argument that was
+provided when the callback was first set. Returning 1 from the callback will
+allow early data and returning 0 will reject it. Note that the
+server may reject early data for other reasons in which case this callback will
+not get called. Notably the built in replay protection feature will still be
```

Comment:
grammar nit: is "built in" with a space really correct here (as opposed to "built-in" or "builtin")?

Reply:
I changed it to "built-in".